# goal
string FileName
geometry_msgs/Pose2D[] FeedbackCallbackPoints
std_msgs/Float32[] ApproachTolerances
std_msgs/Float32[] AttitudeAngleTolerances
bool FeedbackPointAtGoal
std_msgs/Float32 FeedbackPointAtGoalDistance
float32 SpeedMultiplier
float32 k
---
# result
std_msgs/Float32 CurrentVelocity
geometry_msgs/Pose2D LastPoint

---
# feedback
geometry_msgs/Pose2D SelfPos
uint8 CallbackID
